packages/infrastructure/variables.tf

```diff
@@ -0,0 +1,39 @@
+variable "location" {
+  type        = string
+  description = "Primary Location"
+  default     = "westeurope"
+}
+
+variable "name" {
+  type        = string
+  description = "name"
+  default     = "alvtime"
+}
+
+variable "env" {
+  type        = string
+  description = "environment"
+}
+
+variable "aks_service_principal_client_id" {
+  type    = string
+  default = "0.0.0.0"
+}
+
+variable "aks_service_principal_client_secret" {
+  type      = string
+  sensitive = true
+}
+
+variable "sql_server_administrator_login" {
+  type = string
+}
+
+variable "sql_server_administrator_login_password" {
+  type      = string
```

Comment:
Could we create a [key vault](https://azure.microsoft.com/en-us/services/key-vault/) resource using terraform to keep the sql password in?

Reply:
Done